apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,261 +79,163 @@ export function InstructorQuestionSettings({
+                      `
+                    : ''}
+                  ${questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''}
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="qid"
+                    name="qid"
+                    value="${resLocals.question.qid}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    This is a unique identifier for the question. (e.g., "addNumbers")
+                  </small>
+                </div>
+                <div>
+                  <h2 class="h5">Topic</h2>
+                  <div class="row px-3">
+                    <div class="col-1">
+                      ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                        topic: resLocals.topic,
+                      })}
+                    </div>
+                    <div class="col-auto">${resLocals.topic.description}</div>
+                  </div>
+                </div>
+                <hr />
+                <div>
+                  <h2 class="h5">Tags</h2>
+                  ${TagRows({ tags: resLocals.tags })}
+                </div>
+                <hr />
+                <div>
+                  <h2 class="h5">Assessments</h2>
+                  <div>
+                    ${AssessmentRows({
+                      assessmentsWithQuestion,
                     })}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Assessments</th>
-                  <td>
-                    ${resLocals.assessments
-                      ? renderEjs(__filename, "<%- include('../partials/assessments') %>", {
-                          assessments: resLocals.assessments,
-                          urlPrefix: resLocals.urlPrefix,
-                        })
-                      : ''}
-                  </td>
-                </tr>
-                ${sharingEnabled
+                  </div>
+                </div>
+              </form>
+              ${sharingEnabled
+                ? html`
+                    <hr />
+                    <div>
+                      <h2 class="h5">Sharing</h2>
+                      <div data-testid="shared-with">
+                        ${QuestionSharing({
+                          questionSharedPublicly: resLocals.question.shared_publicly,
+                          sharingSetsIn,
+                          hasCoursePermissionOwn: resLocals.authz_data.has_course_permission_own,
+                          sharingSetsOther,
+                          csrfToken: resLocals.__csrf_token,
+                          qid: resLocals.question.qid,
+                        })}
+                      </div>
+                    </div>
+                  `
+                : ''}
+              <hr />
```

Comment:
I'd advise moving this inside the following conditional. As written, we end up with an extra horizontal line for questions that can't be tested.

<img width="262" alt="Screenshot 2024-04-19 at 14 37 21" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/ad1f5619-950c-4450-9991-f8432c051719">

Reply:
Moved the horizontal line into the conditional in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348).